packages/web/babel.config.js

```diff
@@ -0,0 +1,15 @@
+module.exports = function(api) {
+  api.cache(true);
+  return {
+    plugins: [
+      [
+        'module-resolver',
+        {
+          alias: {
+            '^react-native$': 'react-native-web',
```

Comment:
Why is this necessary to add testing-library? create-react-app already does this internally.